src/main/java/domain/Width.java

```diff
@@ -0,0 +1,17 @@
+package domain;
+
+public class Width {
```

Comment:
Width보단 구체적인 네이밍이 좋을 것 같아요.
Width처럼 범용적인 단어를 쓰면 명확한 의도가 안보이고, 다른 곳에서 사용(import)할 때 찾기 불편하더라구요!
```suggestion
public class LadderWidth {
```